packages/SideNavigation/src/components/Item/Item.styles.js

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from "styled-components";
+import Button from "@paprika/button";
+import tokens from "@paprika/tokens";
+import { spacer, fontSize } from "@paprika/stylers/lib/helpers";
+
+export const Item = styled(Button.Link)`
```

Comment:
🤔 so all items on the side menu need to be a button.link? what about those cases where we want to render a div container, or just a regular button that doesn't jump into a new page?

Wouldn't make more sense to set this a `styled.div`

And then you could do `<SideNavigation.Item as={Button.Link} href="">You link?</SideNavigation.Item>`

I guess you can do the opposite like `<SideNavigation.Item as={div} data-type="this is a regular div">My item</SideNavigation.Item>`

If you think `Button.Link` should be the default, then can you add a story pointing out how to render regular items on it?

Reply:
From the current design `Button.Link` looks like the default, sure I'll add a story for rendering custom items!